src/legacy/ui/public/visualize/loader/pipeline_helpers/build_pipeline.ts

```diff
@@ -153,10 +153,6 @@ export const getSchemas = (vis: Vis, timeRange?: any): Schemas => {
   const isHierarchical = vis.isHierarchical();
   const metrics = responseAggs.filter((agg: AggConfig) => agg.type.type === 'metrics');
   responseAggs.forEach((agg: AggConfig) => {
-    if (!agg.enabled) {
-      cnt++;
-      return;
-    }
```

Comment:
Just noticed this -- why was this block removed?

Reply:
It's unreachable code. responseAggs already filtered for agg.enabled. 
in line 152:
`const responseAggs = vis.aggs.getResponseAggs().filter((agg: AggConfig) => agg.enabled);`

Reply:
@lukeelmers, please read my comment to @ppisljar in the review above, i considering to revert my last commit, what do you think?